src/client.ts

```diff
@@ -516,28 +514,16 @@ export class SpvWalletClient {
    * @return {AccessKeys}
    */
   async GetAccessKeys(conditions: AccessKeyFilter, metadata: Metadata, queryParams: QueryParams): Promise<AccessKeys> {
-    return await this.http.request(`access-key/search`, 'POST', {
-      conditions,
-      metadata,
-      page: queryParams?.page || 0,
-      page_size: queryParams?.page_size || 0,
-      order_by_field: queryParams?.order_by_field || '',
-      sort_direction: queryParams?.sort_direction || '',
-    });
-  }
+    const basePath = `users/current/keys`
+    const path = buildPath({
+        filter: conditions,
+        metadata: metadata,
+        queryParams: queryParams,
+        basePath: basePath
+      }
+    );
```

Comment:
@Nazarii-4chain it's a good comment, but we decided with @dorzepowski that it would take some effort to support our filters there + there is specific license in that package.
So we decided that if it works and if it's only one file with ~2 funcs we can write our own implementation.

Reply:
Just for info: There is another package that might help achieve the same
https://www.npmjs.com/package/query-string